modules/admin-quota-overview.adoc

```diff
@@ -0,0 +1,337 @@
+ephemeral storage technology preview. This feature is disabled by default.
+|===
+
+
+.Storage resources managed by quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`requests.storage`
+|The sum of storage requests across all persistent volume claims in any state cannot exceed this value.
+
+|`persistentvolumeclaims`
+|The total number of persistent volume claims that can exist in the project.
+
+|`<storage-class-name>.storageclass.storage.k8s.io/requests.storage`
+|The sum of storage requests across all persistent volume claims in any state that have a matching storage class, cannot exceed this value.
+
+|`<storage-class-name>.storageclass.storage.k8s.io/persistentvolumeclaims`
+|The total number of persistent volume claims with a matching storage class that can exist in the project.
+|===
+
+
+.Object counts managed by quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`pods`
+|The total number of pods in a non-terminal state that can exist in the project.
+
+|`replicationcontrollers`
+|The total number of replication controllers that can exist in the project.
+
+|`resourcequotas`
+|The total number of resource quotas that can exist in the project.
+
+|`services`
+|The total number of services that can exist in the project.
+
+|`secrets`
+|The total number of secrets that can exist in the project.
+
+|`configmaps`
+|The total number of `ConfigMap` objects that can exist in the project.
+
+|`persistentvolumeclaims`
+|The total number of persistent volume claims that can exist in the project.
+
+|`openshift.io/imagestreams`
+|The total number of image streams that can exist in the project.
+
+|===
+
+You can configure an object count quota for these standard namespaced resource types using the `count/<resource>.<group>` syntax.
+
+[source,terminal]
+----
+$ oc create quota <name> --hard=count/<resource>.<group>=<quota> <1>
+----
+
+<1> `<resource>` is the name of the resource, and `<group>` is the API group, if applicable.
+Use the `kubectl api-resources` command for a list of resources and their associated API groups.
+
+== Setting resource quota for extended resources
+
+Overcommitment of resources is not allowed for extended resources, so you must specify `requests` and `limits` for the same extended resource in a quota. Currently, only quota items with the prefix `requests.` are allowed for extended resources. The following is an example scenario of how to set resource quota for the GPU resource `nvidia.com/gpu`.
+
+.Procedure
+
+. To determine how many GPUs are available on a node in your cluster, use the following command:
++
+[source,terminal]
+----
+$ oc describe node ip-172-31-27-209.us-west-2.compute.internal | egrep 'Capacity|Allocatable|gpu'
+----
++
+.Example output
+
+                    openshift.com/gpu-accelerator=true
+Capacity:
+ nvidia.com/gpu:  2
+Allocatable:
+ nvidia.com/gpu:  2
+  nvidia.com/gpu  0           0
```

Comment:
Should be align with above line?

Reply:
@wangke19 Looking now - I am online so I can catch your feedback while you are here!

Reply:
@wangke19  Thank you, I have made this change!